packages/web/src/components/gcds-link/gcds-link.css

```diff
@@ -0,0 +1,43 @@
+:host a {
+  cursor: pointer;
+  color: var(--gcds-link-default);
+
+  background-color: var(--gcds-button-text-only-default-background);
+
+  &
+  :not([aria-disabled='true']) {
+    text-decoration: underline;
+    text-underline-offset: 0.2em;
+    text-decoration-color: currentColor;
+    text-decoration-thickness: var(--gcds-button-text-only-default-decoration-thickness);
+  }
+
+  &
+  :hover {
+    background-color: var(--gcds-button-text-only-default-background);
+    color: var(--gcds-link-hover);
+    text-decoration-thickness: var(--gcds-button-text-only-hover-decoration-thickness);
+  }
+
+  &
+  :focus {
+    background-color: var(--gcds-button-shared-focus-background);
+    color: var(--gcds-button-shared-focus-text);
+    color: var(--gcds-link-light);
+    outline: var(--gcds-button-shared-focus-outline-width) solid var(--gcds-button-shared-focus-background);
+    outline-offset: var(--gcds-button-border-width);
+    border-color: var(--gcds-button-shared-focus-background);
+    text-decoration: none;
+  }
+
+  &
+  :active {
+    background-color: var(--gcds-button-shared-active-background);
+    border-color: var(--gcds-button-shared-active-background);
```

Comment:
Same comment for lines 35 + 36:
Can you replace the button token with a link token instead?